wagtail/models/sites.py

```diff
@@ -196,7 +199,7 @@ def clean_fields(self, exclude=None):
                 )
 
     @staticmethod
-    def get_site_root_paths():
+    def get_site_root_paths() -> List[SiteRootPath]:
```

Comment:
Returning/caching a tuple feels more appropriate than a list here, but I would prefer to keep things backwards-compatible too